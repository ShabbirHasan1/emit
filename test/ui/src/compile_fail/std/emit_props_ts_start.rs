fn main() {
    emit::emit!("template", ts_start: "2024-01-01T00:00:00.000Z");
}
