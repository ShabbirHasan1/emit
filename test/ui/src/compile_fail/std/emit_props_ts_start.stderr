error: specify the start timestamp using the `extent` control parameter before the template
 --> src/compile_fail/std/emit_props_ts_start.rs:2:29
  |
2 |     emit::emit!("template", ts_start: "2024-01-01T00:00:00.000Z");
  |                             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
